renovate.json

```diff
@@ -400,6 +400,32 @@
       "enabled": true,
       "minimumReleaseAge": "7 days"
     },
+    {
+      "groupName": "@elastic/kibana-management dependencies",
+      "matchDepNames": [
+        "@elastic/request-converter",
```

Comment:
`request-converter` has already been added to renovate.json https://github.com/elastic/kibana/blob/main/renovate.json#L1387 but needs to remain separated from this group as it shouln't be backported

Reply:
https://github.com/elastic/kibana/pull/202776/commits/e6a6353ae970a07164213bb008458c1e482d0c2c